### PR TITLE
build(pip): Pin more Google Cloud deps to fix resolution

### PR DESCRIPTION
**Pull request by BYK**
Mirrored these from single-tenant.

### CODE REVIEW

**Review by mitsuhiko — APPROVED** [review]

File: requirements-base.txt [hunk 1]
```diff
@@ -18,10 +18,12 @@ email-reply-parser>=0.2.0,<0.3.0
 enum34>=1.1.6,<1.2.0
 functools32>=3.2.3,<3.3
 futures>=3.2.0,<4.0.0
-# optional
-google-cloud-bigtable>=0.32.1,<0.33.0
 # optional but needed for google stuff below
+google-cloud-bigquery==1.10.0
```
Comment (mattrobenolt) [inline requirements-base.txt]: We shouldn't include bigquery. This has no bearing on open source users.

Comment (mattrobenolt) [reply requirements-base.txt]: It's also arguable that we shouldn't need it in getsentry either. But I'm not sure if we can just yet.

**Comment by mattrobenolt** [comment]
@cameronmcefee is also saying this doesn't fix issues for him.

Can we maybe move to all explicit pins for now for all google libraries until we sort it out?

**Comment by BYK** [comment]
>Can we maybe move to all explicit pins for now for all google libraries until we sort it out?

That'd break getsentry/getsentry compat but I'll do it

**Comment by mattrobenolt** [comment]
I mean to match the pins there.

**Pull request merged**